test/__init__.py

```diff
@@ -100,18 +100,17 @@ def wrapper(*args, **kwargs):
         msg = "Can't run {name} because the network is unreachable".format(
             name=test.__name__)
         try:
-            return test(*args, **kwargs)
+            sock = socket.create_connection((TARPIT_HOST, 80), 0.0001)
+        except socket.timeout:
+            pass
```

Comment:
Because a timeout means the OS found a route, and a route is what we are looking for here.